python/python/glide/async_commands/core.py

```diff
@@ -4288,3 +4360,43 @@ async def object_refcount(self, key: str) -> Optional[int]:
             Optional[int],
             await self._execute_command(RequestType.ObjectRefCount, [key]),
         )
+
+    async def getex(
+        self,
+        key: str,
+        expiry: Optional[ExpiryGetEx] = None,
+    ) -> Optional[str]:
+        """
+        Get the value of `key` and optionally set its expiration. GETEX is similar to GET, but is a write command with `ExpiryGetEx` options.
+        See https://valkey.io/commands/getex for more details.
+
+        Args:
+            key (str): The key to get.
+            expiry (Optional[ExpirySet], optional): set expiriation to the given key.
+                Equivalent to [`EX` | `PX` | `EXAT` | `PXAT` | `KEEPTTL`] in the Redis API. Defaults to None.
```

Comment:
```suggestion
                Equivalent to [`EX` | `PX` | `EXAT` | `PXAT` | `PERSIST`] in the Redis API. Defaults to None.
```

Reply:
What do you mean "Defaults to None?"

Reply:
This was taken from the original ExpirySet. So when the ExpirySet class gets created its pairs of Enum and values, this way it allows PERSIST/KEELTTL to get created with a `null` value is how i interpreted it. It doesnt actually matter what value gets passed because it does not get used in the commands in that case.